### PR TITLE
Update license.

### PR DESCRIPTION
**Pull request by claireh93**
I have updated the license to include myself, Richard and SCRC as copyright holders, as well as tried to fix formatting. I have no idea if this has fixed the ability of GH to automatically detect the license, but perhaps I'll only find that out if I merge back to dev?

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,636 +1,228 @@
...
->   If the disclaimer of warranty and limitation of liability provided
-> above cannot be given local legal effect according to their terms,
-> reviewing courts shall apply local law that most closely approximates
-> an absolute waiver of all civil liability in connection with the
-> Program, unless a warranty or assumption of liability accompanies a
-> copy of the Program in return for a fee.
-> 
->                      END OF TERMS AND CONDITIONS
-> 
+Copyright (c) 2016-2020: Claire Harris, Richard Reeve and the Scottish Covid Response Consortium. This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
```
Comment (richardreeve) [inline LICENSE.md]: COVID-19, not Covid - sorry.

**Review by richardreeve — CHANGES_REQUESTED** [review]
Just a typo.

File: LICENSE.md (hunk 1 above)
Comment (claireh93) [reply LICENSE.md]: Oops! Have fixed

**Review by richardreeve — APPROVED** [review]

**Comment by richardreeve** [comment]
You can just merge this without the tests. There's no possibility it will fail if dev currently passes.

**Comment by richardreeve** [comment]
Closes SCRC-169.

**Pull request merged**